test/integration/logic/BUILD

```diff
@@ -21,57 +21,75 @@ load("@graknlabs_dependencies//builder/java:rules.bzl", "host_compatible_java_te
 host_compatible_java_test(
     name = "test-rule",
     srcs = ["RuleTest.java"],
-    test_class = "grakn.core.logic.RuleTest",
     native_libraries_deps = [
         "//rocks:rocks",
         "//:grakn",
         "//concept:concept",
         "//pattern:pattern",
         "//logic:logic",
     ],
+    resource_strip_prefix = "common/test",
+    resources = [
+        "//common/test:logback",
+    ],
+    test_class = "grakn.core.logic.RuleTest",
     deps = [
         # Internal dependencies
-        "//test/integration/util:util",
-        "//common:common",
+        "//test/integration/util",
+        "//common",
 
         # External dependencies from Grakn Labs
         "@graknlabs_graql//java:graql",
     ],
-    resources = [
-        "//common/test:logback"
-    ],
-    resource_strip_prefix = "common/test",
 )
 
 host_compatible_java_test(
     name = "test-type-hinter",
     srcs = ["TypeHinterTest.java"],
-    test_class = "grakn.core.logic.TypeHinterTest",
+    data = [":basic-schema.gql"],
```

Comment:
does your auto format move these around?? haha data is not source/deps, so we tend to put it last. test_class is super important so it comes towards the top :D

Reply:
I think it must be. I thought I'd installed the Grakn xlm Haikal put on discord to format correctly. Thanks for spotting that.